lib/menu.rb

```diff
@@ -0,0 +1,9 @@
+module Menu
+
+  def menu
+    
+    @takeaway_menu.each do | item, price |
+        puts "#{item}: £#{price}"
+    end
+  end
+end
```

Comment:
Module referring to an instance variable is definitely a code smell - this module can only be included in a class that has a @takeaway_menu  variable. I'm not sure of the advantage of using a module for this. On the other hand, this could be a class that is instantiated and is responsible for collecting menu items.